common/supplemental/supplementalData.xml

```diff
@@ -1272,7 +1272,7 @@ XXX Code for transations where no currency is involved
 		<language type="aa" scripts="Latn"/>
 		<language type="aa" territories="DJ ET" alt="secondary"/>
 		<language type="ab" scripts="Cyrl"/>
-		<language type="ab" territories="GE" alt="secondary"/>
+		<language type="ab" territories="GE TR" alt="secondary"/>
```

Comment:
Hmm, this is data that is generated. To keep this chnage from getting lost we may need the fix discussed in https://unicode-org.atlassian.net/browse/CLDR-13595?focusedCommentId=162212 (and see the comments preceding that for context)

Reply:
@srl295 @pedberg-icu
I am afraid this is going to take more time.
How about we just keep ab_GE, that wouldn't require changes to this file, it's already there.
I'll add ab_TR later.